### ISSUE TITLE
use rotate as the default starting behavior

### ISSUE THREAD

**Issue by wmaiouiru** [publiclab/Leaflet.DistortableImage#51]
Is there any way to use the rotate mode as the default behavior? 
I tried to set the mode to rotate on initialization but had no luck. I also tried to set the mode manually but had no luck either. 

```
im=new L.DistortableImageOverlay(im_url,mode:'rotate').addTo(map);
im.editing._mode='rotate'
```

**Comment by jywarren** [comment]
Hi, @wmaiouiru - there's also been a request for this in the MapKnitter repo, and we're discussing there: publiclab/mapknitter#183

Just curious - what are you using the plugin for? We're very interested in supporting its use in other projects; eager to share and support common use cases.

**Comment by wmaiouiru** [comment]
@jywarren thank you for the quick reply. I am using the plugin for overlaying layers onto maps, and I found the interface quite nice.

**Comment by jywarren** [comment]
Glad to hear it! We're considering making rotate/scale the default starting tool. 

For your code, I believe that you may need to run your second line within a "load" callback, after the image has loaded, as in https://github.com/publiclab/mapknitter/blob/master/app/assets/javascripts/mapknitter/Map.js#L132

Then potentially call `img.fire('update');` to trigger it to update, although I forget if this is necessary.

**Comment by maartenvandillen** [comment]
Thanks for the info, but I could only get this to work by calling the img.editing._toggleRotateDistort() function. Otherwise the handlers were not in sync with the internal mode.

**Comment by sashadev-sky** [comment]
@wmaiouiru @maartenvandillen We heard you guys and have just merged PR #206: which allows a user to pass a `mode` option during `L.distortableImageOverlay` initialization, with the value being the desired default starting editing mode. 

Please checkout the [**Mode** section](https://github.com/publiclab/Leaflet.DistortableImage#mode) in the README for someone additional information on using this option and a code block example.
